test/dotcom_web/controllers/stop_controller_test.exs

```diff
@@ -138,36 +136,23 @@ defmodule DotcomWeb.StopControllerTest do
     test "grouped_route_patterns returns stop's route patterns by route & headsign", %{
       conn: conn
     } do
-      with_mock(RoutePatterns.Repo,
-        by_stop_id: fn "place-here" ->
-          [
-            %RoutePatterns.RoutePattern{
-              route_id: "Purple-A",
-              headsign: "Tree Hill",
-              name: "Here Square - Tree Hill",
-              direction_id: 0
-            }
-          ]
-        end
-      ) do
-        response =
-          get(conn, stop_path(conn, :grouped_route_patterns, "place-here")) |> json_response(200)
-
-        assert %{
-                 "Purple-A" => %{
-                   "Tree Hill" => %{
-                     "direction_id" => 0,
-                     "route_patterns" => [
-                       %{
-                         "headsign" => "Tree Hill",
-                         "name" => "Here Square - Tree Hill"
-                       }
-                       | _
-                     ]
-                   }
+      response =
+        get(conn, stop_path(conn, :grouped_route_patterns, "place-here")) |> json_response(200)
+
+      assert %{
```

Comment:
Is this hard-coded somewhere? This test is a little hard to figure out.

Reply:
I totally forgot about this and because the whole module's excluded it didn't fail the tests! I'll fix this.